### ISSUE TITLE
train is not finished

### ISSUE THREAD

**Issue by tt12354** [marvis/pytorch-yolo3#32]
loss function is still yolov2，do not suit yolov3

**Comment by tt12354** [comment]
Anyone had done this work,please send one to me,I will be very thankful

**Comment by yuyijie1995** [comment]
Can you tell me how to use the code to train my dataset? Thanks

**Comment by tt12354** [comment]
> Can you tell me how to use the code to train my dataset? Thanks
this code is not completed,
